openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -348,19 +348,27 @@ def run(self, *, dry=False, verbose=True,
         equil_time = sim_settings.equilibration_length.to('femtosecond')
         equil_steps = round(equil_time / timestep)
 
+        logger.debug(f'Timestep is {timestep}')
```

Comment:
I don't understand why the logger debug is repeating what's in settings? What are you specifically trying to capture here?

Reply:
I was debugging the rounding of steps to check how small I could make the test.  Yes this is duplicate information from the settings, but it's useful to have here if you want to debug these kinds of things.